app/main/posts/common/post-actions.directive.js

```diff
@@ -51,9 +51,12 @@ function PostActionsDirective(
                     $location.path().indexOf('collections') === -1 &&
                     $location.path().indexOf('savedsearches') === -1) {
                     // Redirect to list
-                    $location.path('/views/data');
+                    $state.go('posts.data');
+                } else if ($state.current.name === 'posts.data' || $state.current.name === 'posts.data.detail') {
+                    // we only want to do this in the data detail view because it's where you can really update a list
+                    $rootScope.$broadcast('event:edit:post:status:data:mode:saveSuccess', {post: $scope.post, deleted: true});
```

Comment:
I suspect we could *always* broadcast the event... and unless we're on the data view there shouldn't be any listeners

Reply:
ah, maybe. I'll check (but, that's depending on the not-using-rootscope thing that I need to check too)